documentation/mdbook.nix

```diff
@@ -11,11 +11,11 @@ base.mkDerivation (attrs // {
     cp -r book/. $out/share/doc/${name}/${type}
   '';
   shellCommands = {
-    run = ''mdbook serve --port ''${1:-3000} "$@" &'';
+    run = {
+      script = ''mdbook serve --port ''${1:-3000} "$@" &'';
+      description = ''
+        Prewview the book and watches the book's src directory for changes, rebuilding the book and refreshing clients for each change.
```

Comment:
Small typo "prewview"

Reply:
:O Spellchecker was not turned on for nix files